### PR TITLE
Attempt to enable downstream tests for holoviews

### PR DESCRIPTION
**Pull request by philippjfr**
Just seeing if I can get this working at first.

- [x] tests added / passed

### CODE REVIEW

**Comment by bryevdv** [comment]
@philippjfr you didn't add `-yes` so the conda install hung on the user prompt. However, the conda install should not go here at all. It should be here:

https://github.com/bokeh/bokeh/blob/master/scripts/ci/install.downstream

**Comment by philippjfr** [comment]
Thanks, didn't really think that through.

**Comment by philippjfr** [comment]
@bryevdv Looks like I messed up specifying the modules in the holoviews 1.10.9 release and the bokeh tests didn't make it into the released package. We'll have a 1.11 release in a week or two, where I'll make sure they are included. Do you mind leaving this open until then or should I reopen this when that's ready?

**Comment by bryevdv** [comment]
@philippjfr what is the status of your 1.10.9 release?

**Comment by philippjfr** [comment]
There never was one unfortunately, we're working hard on getting 1.11 out this week.

**Comment by philippjfr** [comment]
Sorry, there was a 1.10.9 release but that was before I realized I hadn't included the tests.

File: scripts/ci/install.downstream [hunk 1]
```diff
@@ -4,3 +4,4 @@ set -e # exit on error
 set -x # echo commands
 
 conda install --yes --quiet distributed
+conda install --yes --quiet -c pyviz holoviews nose pyviz_comms --no-deps
```
Comment (philippjfr) [inline scripts/ci/install.downstream]:
```suggestion
conda install --yes --quiet -c pyviz/label/dev holoviews nose pyviz_comms --no-deps
```

**Comment by bryevdv** [comment]
@philippjfr looks like its missing param:
```
==================== 37 passed, 2 skipped in 10.54 seconds =====================
+nosetests holoviews/tests/plotting/bokeh
E
======================================================================
ERROR: Failure: ModuleNotFoundError (No module named 'param')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda/lib/python3.6/site-packages/nose/failure.py", line 39, in runTest
    raise self.exc_val.with_traceback(self.tb)
  File "/home/travis/miniconda/lib/python3.6/site-packages/nose/loader.py", line 417, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/travis/miniconda/lib/python3.6/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/travis/miniconda/lib/python3.6/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/travis/miniconda/lib/python3.6/imp.py", line 245, in load_module
    return load_package(name, filename)
  File "/home/travis/miniconda/lib/python3.6/imp.py", line 217, in load_package
    return _load(spec)
  File "<frozen importlib._bootstrap>", line 684, in _load
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/travis/miniconda/lib/python3.6/site-packages/holoviews/__init__.py", line 6, in <module>
    import param
ModuleNotFoundError: No module named 'param'
----------------------------------------------------------------------
Ran 1 test in 0.001s
FAILED (errors=1)
+popd
~/build/bokeh/bokeh
+exit 0
The command "scripts/ci/test" exited with 0.
```

File: scripts/ci/install.downstream [hunk 2]
```diff
@@ -4,3 +4,4 @@ set -e # exit on error
 set -x # echo commands
 
 conda install --yes --quiet distributed
+conda install --yes --quiet -c pyviz/label/dev holoviews nose pyviz_comms --no-deps
```
Comment (philippjfr) [inline scripts/ci/install.downstream]:
```suggestion
conda install --yes --quiet -c pyviz/label/dev holoviews nose
```

File: scripts/ci/install.downstream [hunk 3]
```diff
@@ -4,3 +4,4 @@ set -e # exit on error
 set -x # echo commands
 
 conda install --yes --quiet distributed
+conda install --yes --quiet -c pyviz/label/dev holoviews nose
```
Comment (philippjfr) [inline scripts/ci/install.downstream]:
```suggestion
conda install --yes --quiet -c pyviz/label/dev holoviews nose flexx=0.4.1
```

**Comment by philippjfr** [comment]
Let's wait until tomorrow, I want to make the flexx based formatter tests optional. So I'll release a new dev release and then remove the dependency here.

**Comment by philippjfr** [comment]
Release is now happening tomorrow. We could also merge by pinning to 1.11.0a12 but at this point we might as well wait.

**Comment by bryevdv** [comment]
@philippjfr The buid now fails due to this:
```
PackagesNotFoundError: The following packages are not available from current channels:
  - flexx=0.4.1
```

**Comment by philippjfr** [comment]
@bryevdv I'll resolve any last issues with this today.

**Comment by philippjfr** [comment]
@bryevdv The tests should pass now, the ``distributed`` downstream tests are failing but the 623 holoviews unit tests are passing now. Very glad to finally have this working!

**Comment by bryevdv** [comment]
@philippjfr that failure is almost certainly from the tightening up of `__all__` in `bokeh.models` submodules. Previously, I am sure `field`, `value`, `expr`, etc were transitively imported (unintentionally) There are things I would be willing to say "that was a bug, and this is fixing it" but for these things we should probably just suck it up and explicitly include them in `bokeh.models` (with a comment that they are there to maintain "legacy" API") Do you mind adding that commit to this PR?

**Pull request merged**